plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -718,7 +730,9 @@ if( options.count(op_name) ) { \
 void producer_plugin::plugin_initialize(const boost::program_options::variables_map& options)
 { try {
    my->chain_plug = app().find_plugin<chain_plugin>();
+   my->blockvault_plug = app().find_plugin<blockvault_client_plugin>();
    EOS_ASSERT( my->chain_plug, plugin_config_exception, "chain_plugin not found" );
+   EOS_ASSERT( my->blockvault_plug, plugin_config_exception, "blockvault_client_plugin not found" );
```

Comment:
This should be removed. You don't want to require blockvault_plug to always be available.

Reply:
Change made.